{"uid":"8969aca550a2d01","name":"test_deploy_charms","fullName":"tests.integration.relations.test_relations#test_deploy_charms","historyId":"1caa68ab0aa7f320628ccf9352833a22","time":{"start":1730338558546,"stop":1730338928243,"duration":369697},"description":"Deploy both charms (application and database) to use in the tests.","descriptionHtml":"<p>Deploy both charms (application and database) to use in the tests.</p>\n","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/relations/test_relations.py\", line 63, in test_deploy_charms\n    await ops_test.model.wait_for_idle(apps=APP_NAMES, status=\"active\", timeout=3000)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/0","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1730338558345,"stop":1730338558346,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1730338558346,"stop":1730338558543,"duration":197},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1730338558346,"stop":1730338558346,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1730338558543,"stop":1730338558545,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1730338558545,"stop":1730338558545,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy both charms (application and database) to use in the tests.","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/relations/test_relations.py\", line 63, in test_deploy_charms\n    await ops_test.model.wait_for_idle(apps=APP_NAMES, status=\"active\", timeout=3000)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/0","steps":[],"attachments":[{"uid":"d84976a2b1afd788","name":"log","source":"d84976a2b1afd788.txt","type":"text/plain","size":2655}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1730338928247,"stop":1730338928247,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm::finalizer","time":{"start":1730338928255,"stop":1730338928255,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1730338928255,"stop":1730338928609,"duration":354},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1730338928609,"stop":1730338928609,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1730338928609,"stop":1730338928609,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1730338928609,"stop":1730338928609,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.relations"},{"name":"suite","value":"test_relations"},{"name":"host","value":"github-runner"},{"name":"thread","value":"20745-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.relations.test_relations"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"f89bd1247f61db3b","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":20,"skipped":0,"passed":118,"unknown":5,"total":143},"items":[{"uid":"6e918dc5bea7069c","reportUrl":"../219//#testresult/6e918dc5bea7069c","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1730165643903,"stop":1730165795669,"duration":151766}},{"uid":"98d941450aa2c146","reportUrl":"../218//#testresult/98d941450aa2c146","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1730085642245,"stop":1730085827357,"duration":185112}},{"uid":"5cf6f08347594be1","reportUrl":"../217//#testresult/5cf6f08347594be1","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729992932382,"stop":1729993158717,"duration":226335}},{"uid":"d9821e394d30bb22","reportUrl":"../216//#testresult/d9821e394d30bb22","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729906577868,"stop":1729906794092,"duration":216224}},{"uid":"9324aa01d04fbd74","reportUrl":"../215//#testresult/9324aa01d04fbd74","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729820199283,"stop":1729820467444,"duration":268161}},{"uid":"499eb1a09a6d2643","reportUrl":"../214//#testresult/499eb1a09a6d2643","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729733774696,"stop":1729734027481,"duration":252785}},{"uid":"e2575647258927f2","reportUrl":"../213//#testresult/e2575647258927f2","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729647356333,"stop":1729647629607,"duration":273274}},{"uid":"de497caaa0c595ed","reportUrl":"../212//#testresult/de497caaa0c595ed","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729561049836,"stop":1729561307175,"duration":257339}},{"uid":"4013223340eabe4","reportUrl":"../211//#testresult/4013223340eabe4","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729474441577,"stop":1729474581791,"duration":140214}},{"uid":"c3388b9658c76962","reportUrl":"../210//#testresult/c3388b9658c76962","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729387929714,"stop":1729388126761,"duration":197047}},{"uid":"8f3aaddc88853645","reportUrl":"../209//#testresult/8f3aaddc88853645","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729301757287,"stop":1729301964385,"duration":207098}},{"uid":"7083970b108036e4","reportUrl":"../208//#testresult/7083970b108036e4","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729215228904,"stop":1729215472810,"duration":243906}},{"uid":"3d959cb373ad4cdd","reportUrl":"../207//#testresult/3d959cb373ad4cdd","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729132278862,"stop":1729132406024,"duration":127162}},{"uid":"7addb43474031090","reportUrl":"../206//#testresult/7addb43474031090","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729042603554,"stop":1729042799222,"duration":195668}},{"uid":"fca47ab2ab52fb65","reportUrl":"../205//#testresult/fca47ab2ab52fb65","status":"passed","time":{"start":1728956207925,"stop":1728956548706,"duration":340781}},{"uid":"9f84ba2582cdee05","reportUrl":"../204//#testresult/9f84ba2582cdee05","status":"passed","time":{"start":1728869852472,"stop":1728870196565,"duration":344093}},{"uid":"a58ba17c1736dba9","reportUrl":"../203//#testresult/a58ba17c1736dba9","status":"broken","statusDetails":"juju.errors.JujuAPIError: charm \"ch:amd64/jammy/postgresql-test-app-263\" already exists","time":{"start":1728783378898,"stop":1728783380496,"duration":1598}},{"uid":"47fd3c1167c2e318","reportUrl":"../202//#testresult/47fd3c1167c2e318","status":"passed","time":{"start":1728697018863,"stop":1728697363073,"duration":344210}},{"uid":"e6d359907761a0d2","reportUrl":"../201//#testresult/e6d359907761a0d2","status":"passed","time":{"start":1728610791495,"stop":1728611406130,"duration":614635}},{"uid":"2a2f203d418f78b3","reportUrl":"../200//#testresult/2a2f203d418f78b3","status":"passed","time":{"start":1728524432020,"stop":1728525567654,"duration":1135634}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"8969aca550a2d01.json","parameterValues":[]}